### PR TITLE
chore: added stunner

### PR DESCRIPTION
**Pull request by Icarus131**
Added stunner, as per request in issue #4172

### CODE REVIEW

File: packages/stunner/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# See COPYING for license details.
+
+pkgname=stunner
+pkgver=0.5.8
+pkgrel=1
+pkgdesc='Test and exploit STUN, TURN and TURN over TCP servers.'
+arch=('x86_64' 'aarch64')
+groups=('blackarch' 'blackarch-networking' 'blackarch-webapp')
+url='https://github.com/firefart/stunner'
+license=('CC BY-NC-SA 4.0')
```
Comment (noraj) [inline packages/stunner/PKGBUILD:11]:
```suggestion
license=('CC-BY-NC-SA-4.0')
```

see codenames at https://spdx.org/licenses/

File: packages/stunner/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    -ldflags "-s -w" \
+    -o $pkgname .
+}
+
+package() {
+  cd $pkgname
+
+  install -Dm 755 $pkgname "$pkgdir/usr/bin/$pkgname"
+  install -Dm 644 -t "$pkgdir/usr/share/doc/$pkgname/" README.md
+  install -Dm 644 LICENSE "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
```
Comment (noraj) [inline packages/stunner/PKGBUILD:40]: No need, it's not custom.
```suggestion
```

See https://wiki.archlinux.org/title/PKGBUILD#license

**Review by noraj — CHANGES_REQUESTED** [review]
Thanks for the PR

**Comment by Icarus131** [comment]
Have made changes and pushed @noraj .

**Review by noraj — APPROVED** [review]

**Comment by noptrix** [comment]
thanks a lot

**Pull request merged**